handbook/support.md

```diff
@@ -0,0 +1,13 @@
+# Requesting Support
+
+## Community Forum
+If you have any question like how to do ____ i.e. how to setup SFTP in EasyEngine, you should post in [community forum](https://community.easyengine.io/). There are helpful folks around who might be able to guide you 🕵🏻‍♂️️.
```

Comment:
> i.e. how to setup SFTP in EasyEngine,
This will be covered in docs.

So please make it generic. No example needed.

```suggestion
If you have any question, you should post in [community forum](https://community.easyengine.io/). There are helpful folks around who might be able to guide you 🕵🏻‍♂️️.
```